### PR TITLE
chore(operator): add openebs 2.7.0 operator

### PR DESCRIPTION
**Pull request by akhilerm**
- add openebs operator 2.7.0-(RC1, RC2)
- add ndm operator 1.3.0-(RC1, RC2)
- add cstor operator 2.7.0-(RC1, RC2)
- add zfs operator 1.5.0-(RC1, RC2)
- add lvm oeprator 0.3.0-(RC1, RC2)

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

**Review by shubham14bajpai — APPROVED** [review]
LGTM

**Review by pawanpraka1 — COMMENTED** [review]
- zfs operator yaml looks good.

- We have merged capacity tracking in RC2, we need to fetch the latest opeartor yaml for lvm.

**Comment by akhilerm** [comment]
@pawanpraka1 Updated with the latest changes from lvm-local PV repository

**Comment by pawanpraka1** [comment]
> Updated with the latest changes from lvm-local PV repository

looks good. Please update the RC2 yaml also with lvmnodes CRD.

File: cstor-operator.yaml [hunk 1]
```diff
@@ -1134,7 +1150,9 @@ spec:
               # OpenEBS namespace where the openebs cstor operator components
               # has been installed
             - name: OPENEBS_NAMESPACE
-              value: openebs
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
```
Comment (kmova) [inline cstor-operator.yaml:1155]: cool!

**Comment by akhilerm** [comment]
Done @pawanpraka1

**Review by kmova — APPROVED** [review]

**Pull request merged**